src/Questions/Recaptcha/__tests__/recaptcha.test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import '@testing-library/jest-dom'
+
+import QuestionRecaptcha from '..'
+
+const formDataValues = {}
+const recaptchaRef = React.createRef(null)
+const onSubmitForm = () => {}
+const question = {
+  name: 'recaptcha',
+  type: 'recaptcha',
+  recaptchaKey: 'random'
+}
+
+test('renders a reCAPTCHA', () => {
+  const { getByTestId } = render(
+    <QuestionRecaptcha
+      recaptchaRef={recaptchaRef}
+      formDataValues={formDataValues}
+      onSubmitForm={onSubmitForm}
+      question={question}
+    />
+  )
+
+  expect(getByTestId('recaptcha-test')).toBeTruthy()
```

Comment:
Wondering if we can make the program verify the captcha 🤣

Reply:
To be fair that would disprove its ability to block bots hahaha